presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -93,37 +101,75 @@ Global Properties
 * ``cpuQuotaPeriod`` (optional): the period in which cpu quotas are enforced.
 
 Selectors are processed sequentially and the first one that matches will be used.
-In the example configuration below, there are five resource group templates.
-In the ``adhoc_${USER}`` group, ``${USER}`` will be expanded to the name of the
-user that submitted the query. ``${SOURCE}`` is also supported, which expands
-to the source submitting the query. The source name can be set as follows:
+
+Providing Selector Properties
+-----------------------------
+
+The source name can be set as follows:
 
   * CLI: use the ``--source`` option.
 
   * JDBC: set the ``ApplicationName`` client info property on the ``Connection`` instance.
 
-There are three selectors that define which queries run in which resource group:
+Client tags can be set as follows:
+
+  * CLI: use the ``--client-tags`` option.
+
+  * JDBC: set the ``ClientTags`` client info property on the ``Connection`` instance.
+
+Example
+-------
+
+In the example configuration below, there are several resource groups, some of which are templates.
+Templates allow administrators to construct resource group trees dynamically. For example, in
+the ``pipeline_${USER}`` group, ``${USER}`` will be expanded to the name of the user that submitted
+the query. ``${SOURCE}`` is also supported, which expands to the source submitting the query.
```

Comment:
This tense for this sentence is different from the previous: expanded/expands and submitted/submitting